00-HTML-CSS-basics/ex 1/1.4/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Maximiliano Tulian"/>
+    <meta name="description" content="Exercise 1.4" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Table of expenses</title>
+</head>
+<body>
+    <header>
+        <h1>My Expenses for today</h1>
+    </header>
+    <section>
+        <table summary="This is a table that contains the expenses for today">
+            <caption>Expenses table</caption>
+            <tr>
```

Comment:
It's OK, but maybe you should try to wrap this `<tr>` into a `<thead>`. The same for the other rows, they should be wrapped into `<tbody>`.